lib/formtools/plugins/plugins-helpers.js

```diff
@@ -226,6 +226,41 @@ var overviewDefaults = function (overview, labels) {
 
     overview.summary.label = overview.summary.label || 'Summary';
 
+
+    //overview Details
+    if (!overview.details || typeof overview.details !== 'object') {
+        // defaults to {}
+        overview.details = {};
+    }
+
+    overview.details.form = overviewDetailsDefault(overview.details.fields, schema, form, labels);
+
+    //set tabs to tabsArray property of overview.details
+    var tabsArray = [];
```

Comment:
Could you please rename `tabsArray` to `tabs` and create it via `.map` on a clone of `overview.details.tabs`? For example:

```
var tabs = clone(overview.details.tabs).map(function (tab) {

    // code here...

});
```